Removed the complete field item instance referencing Account.SLAExpirationDate__c; all unrelated FlexiPage metadata remains unchanged.

--- before/backup/flexipages/Person_Account.flexipage-meta.xml
+++ after/working/flexipages/Person_Account.flexipage-meta.xml
@@ -61,16 +61,6 @@
                     <name>uiBehavior</name>
                     <value>none</value>
                 </fieldInstanceProperties>
-                <fieldItem>Record.SLAExpirationDate__c</fieldItem>
-                <identifier>RecordSLAExpirationDate_cField</identifier>
-            </fieldInstance>
-        </itemInstances>
-        <itemInstances>
-            <fieldInstance>
-                <fieldInstanceProperties>
-                    <name>uiBehavior</name>
-                    <value>none</value>
-                </fieldInstanceProperties>
                 <fieldItem>Record.CustomerPriority__c</fieldItem>
                 <identifier>RecordCustomerPriority_cField</identifier>
             </fieldInstance>
@@ -190,4 +180,4 @@
         </properties>
     </template>
     <type>RecordPage</type>
-</FlexiPage>
+</FlexiPage>
\ No newline at end of file
